### PR TITLE
Redirect to the landing page if the user is not logged in

### PR DESCRIPTION
**Pull request by MichaelaTSH**

### CODE REVIEW

File: webproject/src/main/java/com/google/sps/servlets/MyOrderServlet.java
Comment (kevinluvian) [inline webproject/src/main/java/com/google/sps/servlets/MyOrderServlet.java]: does this need to be in this PR?

**Review by kevinluvian — CHANGES_REQUESTED** [review]
will approve after tests are added

File: webproject/src/main/java/com/google/sps/servlets/MyOrderServlet.java
Comment (MichaelaTSH) [reply webproject/src/main/java/com/google/sps/servlets/MyOrderServlet.java]: I've removed it

File: webproject/src/main/java/com/google/sps/servlets/redirect/RedirectServlet.java
Comment (kevinluvian) [inline webproject/src/main/java/com/google/sps/servlets/redirect/RedirectServlet.java]: you should run doGetAuthenticated() directly.
there might be logic in the doGetAuthenticated() that want to redirect to other places, or return the data as json

**Review by kevinluvian — CHANGES_REQUESTED** [review]

File: webproject/src/main/java/com/google/sps/servlets/redirect/RedirectServlet.java
Comment (MichaelaTSH) [reply webproject/src/main/java/com/google/sps/servlets/redirect/RedirectServlet.java]: Done!

File: webproject/src/main/webapp/my-rooms.html
Comment (kevinluvian) [inline webproject/src/main/webapp/my-rooms.html]: is this link correct?

**Review by kevinluvian — CHANGES_REQUESTED** [review]

File: webproject/src/main/webapp/js/room-my-order.js
Comment (kevinluvian) [inline webproject/src/main/webapp/js/room-my-order.js]: is this supposed to be deleted?

**Review by kevinluvian — CHANGES_REQUESTED** [review]

File: webproject/src/main/webapp/my-rooms.html
Comment (kevinluvian) [inline webproject/src/main/webapp/my-rooms.html]: is this file exists?

**Review by kevinluvian — CHANGES_REQUESTED** [review]

File: webproject/src/main/webapp/js/room-my-order.js
Comment (MichaelaTSH) [reply webproject/src/main/webapp/js/room-my-order.js]: Thank you so much for spotting this, I hadn't realised! It was deleted because when I made this redirect PR, I mistakenly included a separate Delete Order PR inside this PR before the Delete Order PR was merged. I did not think that after the Delete Order PR was merged, it would delete as well. I will fix this!

Comment (MichaelaTSH) [reply webproject/src/main/webapp/js/room-my-order.js:122]: Thank you so much for spotting this, I hadn't realised! It was deleted because when I made this redirect PR, I mistakenly included a separate Delete Order PR inside this PR before the Delete Order PR was merged. I did not think that after the Delete Order PR was merged, it would delete as well. I will fix this!

Comment (MichaelaTSH) [reply webproject/src/main/webapp/js/room-my-order.js:122]: Thank you so much for spotting this, I hadn't realised! It was deleted because when I made this redirect PR, I mistakenly included a separate Delete Order PR inside this PR before the Delete Order PR was merged. I did not think that after the Delete Order PR was merged, it would delete as well. I will fix this!

File: webproject/src/main/webapp/my-rooms.html
Comment (MichaelaTSH) [reply webproject/src/main/webapp/my-rooms.html]: No, it does not! I did try to delete all of the references but it seems I missed one.

File: webproject/src/main/webapp/create-room.html
Comment (kevinluvian) [inline webproject/src/main/webapp/create-room.html]: theres still the file here

**Review by kevinluvian — APPROVED** [review]

File: webproject/src/main/webapp/create-room.html
Comment (MichaelaTSH) [reply webproject/src/main/webapp/create-room.html]: Right

**Pull request merged**
